From 3ad93913ebddad103583a42adfbf0858727ae1c3 Mon Sep 17 00:00:00 2001
From: Wang Panzhenzhuan <randy.wang@rock-chips.com>
Date: Tue, 3 Mar 2020 15:58:26 +0800
Subject: [PATCH] ARM: rockchip_defconfig: rk3288 evb board: enable ov13850 &
 vcm149c & sgm3784

ov13850 camera module used by rk3288 evb board

Signed-off-by: Wang Panzhenzhuan <randy.wang@rock-chips.com>
Change-Id: I23ad3dcb06a9f5c748a9399f5d4e9e9b1936ca52
---
 arch/arm/configs/rockchip_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index c3113fa2828a..04fc4a57e068 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -465,10 +465,13 @@ CONFIG_V4L_PLATFORM_DRIVERS=y
 CONFIG_VIDEO_ROCKCHIP_CIF=y
 CONFIG_VIDEO_ROCKCHIP_ISP1=y
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
+CONFIG_VIDEO_OV13850=y
 CONFIG_VIDEO_GC0312=y
 CONFIG_VIDEO_GC0329=y
 CONFIG_VIDEO_GC2035=y
 CONFIG_VIDEO_GC2145=y
+CONFIG_VIDEO_VM149C=y
+CONFIG_VIDEO_SGM3784=y
 CONFIG_DRM=y
 CONFIG_DRM_IGNORE_IOTCL_PERMIT=y
 CONFIG_DRM_LOAD_EDID_FIRMWARE=y
-- 
2.35.3

